editor/export/editor_export_platform.cpp

```diff
@@ -141,7 +141,9 @@ void EditorExportPlatform::gen_debug_flags(Vector<String> &r_flags, int p_flags)
 	String host = EDITOR_GET("network/debug/remote_host");
 	int remote_port = (int)EDITOR_GET("network/debug/remote_port");
 
-	if (p_flags & DEBUG_FLAG_REMOTE_DEBUG_LOCALHOST) {
+	if (EDITOR_GET("export/android/use_wifi_for_remote_debug")) {
+		host = EDITOR_GET("export/android/wifi_remote_debug_host");
```

Comment:
I'm a bit concerned about that check, as it would fail if `export/android/use_wifi_for_remote_debug` doesn't exist. This could happen if someone does a custom build without Android support for one reason or another. The platform code is meant to stay modular so that it can be easily swapped for other platforms. In this case it's unlikely to ever be a problem for Android, but this slightly breaks the encapsulation.

Technically, this check would not be needed if the Android remote debug host was configured via `network/debug/remote_host`. Is there any issue with this approach, aside from it separating the setting to force override WiFi debug, and the actual debug configuration?

Reply:
CC @bruvzg

Reply:
The reason I didn't use `network/debug/remote_host` is because currently that field is a dropdown which only gives the options for internal IPs assigned to the computer running the editor. For my use-case I need to put in the external IP as we sometimes walk out of the range of the LAN while testing

Reply:
I guess we can go ahead with the current approach, but you should check that the settings exist otherwise it will show an error if for whatever reason Godot is compiled with the Android platform:
```
if (EditorSettings::get_singleton()->has_setting("export/android/use_wifi_for_remote_debug") && EDITOR_GET("export/android/use_wifi_for_remote_debug")) {
```